src/CommunityCommons/javasource/unittesting/RemoteApiServlet.java

```diff
@@ -6,7 +6,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 import org.apache.commons.io.IOUtils;
-import org.apache.hc.client5.http.auth.InvalidCredentialsException;
+import org.apache.http.auth.InvalidCredentialsException;
```

Comment:
Please note you're making a change to the [unittesting module](https://github.com/mendix/UnitTesting/blob/e60867b3fb3809af71799c6725d96cc8b41d70ef/src/javasource/unittesting/RemoteApiServlet.java#L9) code here, which will be overwritten if the module is ever upgraded.

Reply:
Yep, that works.